src/macros.rs

```diff
@@ -104,12 +104,13 @@ macro_rules! py_run_impl {
     }};
     ($py:expr, *$dict:expr, $code:expr) => {{
         use ::std::option::Option::*;
-        if let ::std::result::Result::Err(e) = $py.run($code, None, Some($dict)) {
+        use $crate::PyNativeType;
```

Comment:
I'm a little surprised that this doesn't need an `#[allow(unused_imports)]` 🤔. Eh, we can add it later if it becomes a problem.